src/style.css

```diff
@@ -1 +1,105 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  width: 100%;
```

Comment:
there is no need to write width on your parent block cause this block will stretch by child elements

also you should try to write the BEM block (in our case form) in such a way that your block will not be broken if you move it to another place.

therefore, ideally, the sizes and positioning of blocks should not be explicitly specified. it is better to explicitly set the width-height to the child elements (in our case input elements) and form block are stretched automatically